multi-language-examples/README.rst

```diff
@@ -0,0 +1,112 @@
+.. Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
+   International License (the "License"). You may not use this file except in compliance with the
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+
+Welcome
+=======
+This part of the AWS Code Example Repository contains code samples that span multiple programming
+languages. The examples use the AWS Software Developer Kits (SDKs) for the supported programming
+languages. For more information, see `Tools to Build on AWS 
+<https://aws.amazon.com/getting-started/tools-sdks/>`_.
+
+Finding code examples
+=====================
+
+The code examples in this folder are organized by the service that the example demonstrates. If an
+example involves multiple services, then you'll find it in the folder `multi-service-examples 
+<mult-service-examples>`_. programming language.
+
+You can also try using the preview of our `use case index of examples <code-index.md>`_ (and let us
+know what you think about it).
+
+Building and running code examples
+==================================
+
+Inside each of the language-specific directories, you'll find a **README** file that explains how to
+build and run the examples contained within it.
+
+The example code inside the language-specific directories is organized by
+the AWS service abbreviation ("s3" for `Amazon S3 <https://aws.amazon.com/s3>`_ examples, and so
+on).
+
+Proposing new code examples
```

Comment:
Isn't this the same text as in the parent readme? Can't we just refer them to that file instead of repeating ourselves?

Reply:
I'll examine and consider doing that.